zulipterminal/helper.py

```diff
@@ -477,6 +477,14 @@ def canonicalize_color(color: str) -> str:
         raise ValueError('Unknown format for color "{}"'.format(color))
 
 
+def styled_typeahead(typeaheads: List[str],
+                     wrapping_text: str) -> List[str]:
```

Comment:
I assume this method is a precursor to make styled (with bugdown/markdown) versions separate from the versions in the bottom bar? If so, this feels like it belongs with that work.

We could include it now, but it's clearer seeing it 'in action' - demonstrating why it's much better - though arguably it does that to some extent already.

Reply:
Right, this is mainly to decrease the clutter in the footer PR. It'll be better if I put it there :+1: